### ISSUE TITLE
I want something to be bold, do I use <b> or <em> or some css or what?

### ISSUE THREAD

**Issue by renoirb** [renoirb/htmlcsstherightway#5]
Source: https://github.com/h5bp/lazyweb-requests/issues/73

**Comment by pankajparashar-zz** [comment]
The best way to style text is by using CSS. Hence, style="font-weight:bold" is my recommendation.

**Comment by renoirb** [comment]
You realize that thee goal of that bug tracker list is to document things a person who wants to start would read and learn?  If you want to suggest content, not only one liners ;)

**Comment by viditvarshney** [comment]
Yes as @pankajparashar-zz  says best way to bold text is by CSS 
style = "font-weight: bold"

for a part of text you can also use <b> or <em> also
